crates/fuel-core/src/service/genesis/exporter.rs

```diff
@@ -84,7 +86,9 @@ where
             ContractsLatestUtxo,
             ContractsState,
             ContractsAssets,
-            Transactions
+            Transactions,
+            FuelBlocks,
+            SealedBlockConsensus
```

Comment:
I've just realized that while it works for the first-time regenesis, it will not work for the next regenesises because it ignores `Old*` tables.

We need to migrate(export and import) them as well as part of the off-chain snapshot.

You have two variants, you can export and import all `Old*` tables, and it should work. Or, during exporting instead of exporting them into `Transactions`/`FuelBlocks`/`SealedBlockConsensus`, you can export directly into `OldTransactions/`OldFuelBlocks`/`OldFuelBlockConsensus`.